src/views/wallet/Transfer.vue

```diff
@@ -497,15 +500,11 @@ export default class Transfer extends Vue {
             unsignedTx.fromBuffer(Buffer.from(this.pendingSendMultisigTX.tx?.unsignedTx, 'hex'))
             const utx = unsignedTx.getTransaction()
             this.memo = utx.getMemo().toString()
-            const toAddress = WalletHelper.getToAddressFromUtx(
-                unsignedTx,
-                this.wallet.getStaticAddress('P')
-            )
-            // eslint-disable-next-line no-control-regex
-            if (toAddress) this.addressIn = 'P' + toAddress?.replace(/\x00/g, '')
-            this.totalAmount = toAddress
-                ? WalletHelper.getTotalAmountFromUtx(unsignedTx, toAddress)
-                : undefined
+            let t: ITransactionData = parse([this.pendingSendMultisigTX])[0]
```

Comment:
You can wrap with a try/catch in this case because the value might be undefined

Reply:
the parse function always returns ITransactionData it cant be undefined